lightgbm/src/test/scala/com/microsoft/azure/synapse/ml/lightgbm/split1/VerifyLightGBMClassifier.scala

```diff
@@ -185,13 +185,16 @@ class VerifyLightGBMClassifier extends Benchmarks with EstimatorFuzzing[LightGBM
     assertBinaryImprovement(scoredDF1, scoredDF2)
   }
 
-  ignore("Verify LightGBM Multiclass Classifier with vector initial score") {
-    val scoredDF1 = baseModel.fit(breastTissueDF).transform(breastTissueDF)
-    val df2 = scoredDF1.withColumn(initScoreCol, col(rawPredCol))
-      .drop(predCol, rawPredCol, probCol, leafPredCol, featuresShapCol)
-    val scoredDF2 = baseModel.setInitScoreCol(initScoreCol).fit(df2).transform(df2)
-
-    assertMulticlassImprovement(scoredDF1, scoredDF2)
+  test("Verify LightGBM Multiclass Classifier with vector initial score") {
```

Comment:
might be a good idea to add a test just for the insertTransposedChunkedArray to validate it, but not necessary for this PR